conf/test_route_control.py

```diff
@@ -244,7 +279,9 @@ def test_given_valid_new_route_when_add_new_route_entry_and_mac_known_and_neighb
             prefix_len=24,
         )
         self.route_controller.add_new_route_entry(route_entry)
+        self.mock_bess_controller.create_module()
         self.mock_bess_controller.create_module.assert_called()
+        self.mock_bess_controller.link_modules()
         self.mock_bess_controller.link_modules.assert_called()
```

Comment:
What is your suggestion to address this? just deleting these 4 lines? Or am I misunderstanding your comment? Without the `.create_module()` and `.link_modules()`, the test failed

Reply:
```suggestion
        self.mock_bess_controller.create_module.assert_called()
        self.mock_bess_controller.link_modules.assert_called()
```
I would remove those lines, and change the `kwards['dst']` above in the test (left a suggestion for the exact change). This will make the test pass as it will find the mac address for the next hop and call bess to add the modules.